.github/workflows/unit-tests.yaml

```diff
@@ -40,4 +40,4 @@ jobs:
     - name: CloudFormation lint test
       run: cfn-lint code/solutions/*.yaml
     - name: CloudFormation nag test
-      run: cfn_nag_scan --input-path code/solutions
+      run: cfn_nag_scan --input-path code/solutions --ignore-fatal
```

Comment:
I don't think we want this

Reply:
I believe `--ignore-fatal` helps to fix issue when cfn_nag tries to process non CloudFormation files. Found in `cfn_nag --help` 

```
-n, --ignore-fatal    Ignore files with fatal violations.  
    Useful for ignoring non-Cloudformation yaml/yml/json in a path
```

Reply:
Adopting `.template` will fix this too :)

Reply:
In the `cfn_nag` usage output, I read a note for the `--ignore-fatal` option: _"[...] Useful for ignoring non-Cloudformation yaml/yml/json in a path"_.  @rezabekf - I believe this was the intent here?